yt/frontends/enzo_e/data_structures.py

```diff
@@ -406,9 +406,12 @@ def _parse_parameter_file(self):
         ablock = fh[list(fh.keys())[0]]
         self.current_time = ablock.attrs["time"][0]
         self.parameters["current_cycle"] = ablock.attrs["cycle"][0]
-        gsi = ablock.attrs["enzo_GridStartIndex"]
+        gsi = ablock.attrs["enzo_GridStartIndex"]  # <- always has 3 elements
         gei = ablock.attrs["enzo_GridEndIndex"]
-        self.ghost_zones = gsi[0]
+        # Enzo-E technically allows each axis to have different ghost zone
+        # depths (this feature is not really used in practice)
+        self.ghost_zones = gsi
```

Comment:
although technically not backward compatible, I don't expect anyone to rely on the old behavior so I'm tempted to just let this change in.

Reply:
I definitely agree, especially because pool of enzo-e users is currently very small. (At most, there are maybe 5 people who regularly use the yt-frontend right now)

But, let me know if you would prefer I change it (and whether there is an obvious preferred way to do it)

Reply:
I think it's fine as is. I'll just give it a couple weeks before merging in case anyone objects.